doc-pages/.eslintrc

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Planning on unifying things for the entire code base if possible - this is to clean this codebase up first.